internal/tiltfile/docker_compose.go

```diff
@@ -426,7 +445,11 @@ func (s *tiltfileState) dcServiceToManifest(service *dcService, dcConfigPath str
 
 	dcInfo = dcInfo.WithBuildPath(service.BuildContext)
 
-	paths := []string{path.Dir(service.DfPath), path.Dir(dcConfigPath)}
+	var paths []string
```

Comment:
```suggestion
	paths := []string{path.Dir(service.Dfpath)}
```

Then you can delete L452.